raft/server.cc

```diff
@@ -1469,13 +1480,14 @@ future<read_barrier_reply> server_impl::execute_read_barrier(server_id from, sea
     logger.trace("[{}] execute_read_barrier read id is {} for commit idx {}",
         _id, rid->first, rid->second);
     if (as && as->abort_requested()) {
-        return make_exception_future<read_barrier_reply>(request_aborted());
+        return make_exception_future<read_barrier_reply>(
+            request_aborted(format("Abort requested while executing read barrier, read id is {} for commit idx {}", rid->first, rid->second)));
```

Comment:
Similar comment to the one in `wait_for_entry`.

Here we could say "before waiting for read barrier" (because "before executing" would strictly speaking be incorrect; we start executing it above in the background by the `_fsm->start_read_barrier`, but we didn't start waiting for it), and below say "while waiting for read barrier".